nemo/collections/common/tokenizers/text_to_speech/ipa_lexicon.py

```diff
@@ -23,6 +23,13 @@
     ')', '[', ']', '{', '}',
 )
 
+VITS_PUNCTUATION = (
+    ',', '.', '!', '?', '-',
+    ':', ';', '"', '«', '»',
+    '“', '”', '¡', '¿', '—', 
+    '…',
+)
+
```

Comment:
Could you pls document why introducing this new `VITS_PUNCTUATION`? Are these new punctuations used by other languages?